src/hydra-evaluator/hydra-evaluator.cc

```diff
@@ -13,7 +13,57 @@
 
 using namespace nix;
 
-typedef std::pair<std::string, std::string> JobsetName;
+typedef std::pair<std::string, std::string> JobsetSymbolicIdentity;
+
+class JobsetIdentity {
```

Comment:
Maybe just call this `JobsetId` to be less verbose?

Reply:
Good idea. I went verbose to make the refactor easier, cleaning up makes sense. :+1: